packages/css/src/formElements/radiobutton/radiobutton.css

```diff
@@ -28,6 +28,11 @@
   cursor: pointer;
 }
 
+.md-radiobutton__input {
+  width: 1.45rem;
```

Comment:
Kunne lagt til width og height på .md-radiobutton input[type='radio']-selectorn her over for å slippe lage nye klasse

Reply:
Ok, takk! Da gjør jeg det.

Reply:
Oops, så ikke den. Flaut.